docs/production-setup.md

```diff
@@ -201,13 +270,15 @@ docker run --rm hermeznetwork/zkevm-node:latest sh -c "/app/zkevm-node encryptKe
 
 The command above will create the file `acc.keystore` inside of the `zkevm-node` directory.
 
-After it we need to create a configuration file to provide the configurations to the node, to achieve this create a file called `config.toml` inside of the `zkevm-node` directory, then go to the example [config file](config/config.debug.toml) and `copy/paste` the content into the `config.toml` you'll actually use.
+After it we need to create a configuration file to provide the configurations to the node, to achieve this create a file called `config.toml` inside of the `zkevm-node` directory, then go to the example [config file](./config/environments/public/public.node.config.toml) (`./config/environments/public/public.node.config.toml`) and `copy/paste` the content into the `config.toml` you'll actually use.
+
+Do the same for the `genesis` file: [genesis file](./config/environments/public/public.genesis.config.json) (`./config/environments/public/public.genesis.config.json`)
 
 Remember to:
 
 - replace the database information if you set it differently while setting up the Postgres instance
 - set the `Database Host` with the `Postgres instance IP`
-- set the `Etherman URL` with the `JSON RPC URL` of the `Ethereum node`
+- set the `Etherman URL` with the `JSON RPC URL` of the `Ethereum node` you created earlier *or* use the Infura URL with your API key like so: `https://goerli.infura.io/v3/$YOUR_API_KEY`
 - set the `Etherman Password` to allow the node to decrypt the `keystore file`
```

Comment:
I think we don't need this anymore.

Reply:
What exactly don't we need? Rolling the Ethereum node manually, right?

Reply:
@tclemos do I remove the section for creating an L1 Geth node?